po/extra/pt.po

```diff
@@ -0,0 +1,360 @@
+# Portuguese translations for extra package.
+# Copyright (C) 2020 THE extra'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the extra package.
+# Automatically generated, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-12-04 13:13+0900\n"
+"PO-Revision-Date: 2021-01-02 18:02+0000\n"
+"Last-Translator: Hugo Carvalho <hugokarvalho@hotmail.com>\n"
+"Language-Team: Portuguese (pt)\n"
+"Language: pt\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 2.4.2\n"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:7
+#: data/com.github.donadigo.eddy.desktop.in:4
+msgid "Eddy"
+msgstr "Eddy"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:8
+msgid "Install and Manage .deb Files"
+msgstr "Instalar e gerir ficheiros .deb"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:10
+msgid "Install, uninstall and view information about debian packages."
+msgstr "Instalar, desinstalar e ver informação sobre pacotes debian."
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:11
+msgid "Features include:"
+msgstr "Funcionalidades incluídas:"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:13
+msgid "Install multiple packages at once"
+msgstr "Instala vários pacotes ao mesmo tempo"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:14
+msgid "Upgrade and downgrade existing packages"
+msgstr "Atualiza e desatualiza de pacotes existentes"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:15
+msgid "Load all packages automatically from your Downloads folder"
+msgstr ""
+"Carregue todos os pacotes automaticamente a partir da sua pasta de "
```

Comment:
For the verb tenses to be consistent, this should say "Carrega".

Reply:
Thanks for the tip.
I'll fix that.